ext/php7/serializer.c

```diff
@@ -570,6 +571,10 @@ void ddtrace_set_root_span_properties(ddtrace_span_t *span) {
         }
     }
 
+    if (!get_DD_TRACE_CLIENT_IP_HEADER_DISABLED()) {
+        ddtrace_extract_ip_from_headers(&PG(http_globals)[TRACK_VARS_SERVER], meta);
```

Comment:
This needs a guard whether this is indeed an array - there's nothing preventing the user from redefining these...